tests/test_viewsets.py

```diff
@@ -286,4 +296,22 @@ def test_map_claim_to_model(self):
             # Redirects to the redirect url on successful user creation
             self.assertEqual(response.status_code, 302)
             # There has been no change in number of User accounts
+            self.assertEqual(User.objects.count(), user_count)
+
+            # Name is split into first_name and last_name if both any is not
+            # present
+            mock_func.return_value = {
+                "given_name": "Wrong",
+                "name": "Davis Raym",
+                "sub": "davis@m.com",
+            }
+            data = {"id_token": "saasdrrw.fdfdfdswg4gdfs.sadadsods"}
```

Comment:
Is this id token value just used for testing?
Should this be in the settings file?

Reply:
This is just a random mocked value; I don't think it should be in the settings